SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -0,0 +1,326 @@
+|pm:PatientContextState
+/pm:CoreData
+/pm:Middlename
+|
+
+|PID-5/XPN-5
+|Prefix (e.g. DR)
+|pm:PatientContextState
+/pm:CoreData
+/pm:Title
+|
+
+|PID-5/XPN-7
+|Name Type Code
+|pm:PatientContextState
+/pm:CoreData
+|This field shall be set to "L" when a patient name is available, or "U" when the patient name is not set.
+
+Please refer also to the corresponding section in the <<ihe_pcd_tf-2_2019>>.
+
+|===
+
+====== PID-6 Mother’s Maiden Name
+The PID-6 field shall contain the mother's maiden name or birth name before marriage. The table <<ref_tbl_pid6_mapping>> defines the mapping of the SDC patient name information to the data fields of the HL7 data type *XPN* used in the PID-6 field.
+
+[#ref_tbl_pid6_mapping]
+.PID-6 Mother’s Maiden Name Mapping
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|PID-6/XPN-1
+|Family Name
+|pm:PatientContextState
+/pm:CoreData
+|HL7 data type *FN*
+
+|PID-6/XPN-1.1
+|Surname
+|/pm:Birthname
+|
+
+|===
+
+====== PID-7 Date/Time of Birth
+The PID-7 field shall contain the date & time of birth. The table <<ref_tbl_pid7_mapping>> defines the mapping of the SDC patient's date of birth information to the data fields of the HL7 data type *DTM* used in the PID-7 field.
+
+[#ref_tbl_pid7_mapping]
+.PID-7 Date/Time of Birth Mapping
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|PID-6/DTM-1
+|Date/Time
+|pm:PatientContextState
+/pm:CoreData
+/pm:DateOfBirth
+|Note that the HL7 date & time format differs from the xsd date/time formats and requires a mapping accordingly (see also <<ref_expl_dt_mapping>>).
+
+|===
+
+[#ref_expl_dt_mapping]
+.Date/Time Format Mapping
+====
+xsd:dateTime: *2001-10-26T21:32:52* -> HL7 DTM: *20011026213252*
+
+xsd:date: *2001-10-26* -> HL7 DTM: *20011026*
+====
+
+====== PID-8 Administrative Sex
+The PID-8 field shall contain the code for the administrative sex. However, in the clinical context of a point of care device the biological sex, or sex at birth is important for various algorithms. The table <<ref_tbl_pid8_mapping>> defines the mapping of the SDC patient's sex information to the data fields of the HL7 data type *IS* used in the PID-8 field.
+
+[NOTE]
+====
+The sex and gender of a patient (or a newborn) cannot exactly be mapped from ISO/IEEE 11073-10207 to HL7 V2. The domain information and service model only contains an attribute for sex as defined by biological and physiological characteristics. HL7 V2, on the other hand, only provides a field for the administrative sex as defined by the socially constructed roles, behaviours, activities, and attributes that a given society considers appropriate. The biological sex, however, does not necessarily match a person’s administrative gender. Mapping from one to the other would therefore introduce errors. However, in the clinical context of a point of care device the biological sex, or sex at birth is important for various algorithms, and therefore, shall be mapped to the PID-8 field.

```

Comment:
Didn't we decide at the workshop that this field could not be mapped? Even though gender is very important for clinical applications, I don't think a mapping is appropriate here.

Reply:
Yes, you are right. However, the sex used by the device e. g. for the ECG algorithm might be important for the system receiving the data. When the data are not plausible or lead to wrong diagnosis, the clinician could still check which sex has been set during the measurement. I think this is an important information. It must be clear for the receiver that the sex from the gateway is a setting on the device used for clinical applications.

Reply:
As written in ISO 27269 International Patient Summary: 
"Some countries require 'gender' as part of their identification of the patient. It SHALL not be used to record a patient's sex.", I do not believe this mapping is appropriate. I suggest using an OBX  segment with the LOINC Code 99501-9 and the new HL7 FHIR Valueset https://build.fhir.org/extension-patient-sexforclinicaluse.html

Reply:
Todd recommended to add a note for the reviewers to asked for their opinions.

Reply:
For the PID-8 mapping, I added a note for reviews with a request for comment.